axelor-human-resource/src/main/java/com/axelor/apps/hr/web/timesheet/QuickTimesheetController.java

```diff
@@ -0,0 +1,169 @@
+
+      ActionView.ActionViewBuilder actionViewBuilder =
+          ActionView.define(I18n.get("Timesheet lines"))
+              .model(TimesheetLine.class.getName())
+              .add("grid", "quick-timesheet-line-grid")
+              .add("form", "quick-timesheet-line-form")
+              .param("popup", "true")
+              .param("popup-save", "true")
+              .param("forceEdit", "true")
+              .domain("self.employee = :employee AND self.date between :fromDate and :toDate")
+              .context("employee", user.getEmployee())
+              .context("fromDate", LocalDate.parse(fromDateContext.toString()))
+              .context("toDate", LocalDate.parse(toDateContext.toString()));
+
+      response.setView(actionViewBuilder.map());
+    }
+  }
+
+  @SuppressWarnings("unchecked")
+  public void computeTotals(ActionRequest request, ActionResponse response) throws AxelorException {
+
+    Object userContext = request.getContext().get("user");
+    Object fromDateContext = request.getContext().get("fromDate");
+    Object toDateContext = request.getContext().get("toDate");
+    Object timeUnitContext = request.getContext().get("timeLoggingPreferenceSelect");
+
+    if (userContext != null
+        && fromDateContext != null
+        && toDateContext != null
+        && timeUnitContext != null) {
+
+      User user =
+          userContext instanceof User
+              ? (User) userContext
+              : Beans.get(UserRepository.class)
+                  .find(
+                      Long.valueOf(
+                          ((LinkedHashMap<String, Object>) userContext).get("id").toString()));
+
+      Employee employee = user.getEmployee();
+
+      if (employee != null) {
+        LocalDate fromDate = LocalDate.parse(fromDateContext.toString());
+        LocalDate toDate = LocalDate.parse(toDateContext.toString());
+        String timeUnit = (String) timeUnitContext;
+
+        QuickTimesheetService quickTimesheetService = Beans.get(QuickTimesheetService.class);
+
+        BigDecimal totalTimeEntriesForPeriod =
+            quickTimesheetService.computeTotalTimeEntriesForPeriod(employee, fromDate, toDate);
+        BigDecimal totalLeavesAndHolidaysForPeriod =
+            quickTimesheetService.computeTotalLeavesAndHolidaysForPeriod(
+                employee, fromDate, toDate, timeUnit);
+        BigDecimal totalWorkDaysForPeriod =
+            quickTimesheetService.computeTotalWorkDaysForPeriod(
+                employee, fromDate, toDate, timeUnit);
+
+        response.setValue("totalTimeEntriesForPeriod", totalTimeEntriesForPeriod);
+        response.setValue("totalLeavesAndHolidaysForPeriod", totalLeavesAndHolidaysForPeriod);
+        response.setValue(
+            "totalDueTimeEntriesForPeriod",
+            totalWorkDaysForPeriod
+                .subtract(totalTimeEntriesForPeriod)
+                .subtract(totalLeavesAndHolidaysForPeriod));
+      }
```

Comment:
Remove the leaves and holiday in compute of the last field because it's already substract in the method